### ISSUE TITLE
Windows: reduce flickering when resizing

### ISSUE THREAD

**Issue by tobtoht** [monero-project/monero-gui#2657]
See: http://xmrguide42y34onq.onion/static/flickering.mp4 (Tor required)

**Comment by selsta** [comment]
Are you using a VM?

**Comment by tobtoht** [comment]
No, not a VM. Tested with v0.15.0.2 release binary running Windows 10 on real hardware.

>Might be solved once we move to newer Qt version.

Ok, I'll keep an eye out for this.

**Comment by selsta** [comment]
As a workaround disabling custom decorations should solve the flickering issue.

We have our own resizing code when custom decorations are enabled which isn’t optimized.

To properly solve the issue we would have to write custom code for every OS to disable window decorations without disabling resizing.
